runtime/codert_vm/jswalk.c

```diff
@@ -1225,6 +1225,7 @@ static void jitWalkResolveMethodFrame(J9StackWalkState *walkState)
 		while ((sigChar = jitNextSigChar(&sigData)) != ')') {
 
 			switch (sigChar) {
+				case 'Q': /* fall through */
```

Comment:
You may want to fix the same code in `JITStackWalker.java` as well.

Reply:
updated in 888e0d0